commands/plugin-install.js

```diff
@@ -35,7 +35,15 @@ module.exports = async ({ configuration, serviceDir, configurationFilename, opti
     { isMainEvent: true }
   );
   await installPlugin(context);
-  await addPluginToServerlessFile(context);
+  // Check if plugin is already added
+  const pluginAlreadyPresentInConfig =
+    (configuration.plugins &&
+      configuration.plugins.modules &&
```

Comment:
nit: We may use `_.get(configuration, 'plugins.modules')` instead. 

Using it, will make future refactor to optional chaining (`confiugration.?plugins.?modules`) really easy, as we can find all occurrences of `_.get` and simply change them.

We would be able to do it with next major, when we will drop support for Node.js v12

Reply:
Sure, good idea 👍